src/components/layout/Templates/BoardLayout.tsx

```diff
@@ -243,6 +243,12 @@ const BackgroundImage = () => {
     return null;
   }
 
+  // Check if the background image URL is a video
+  const videoFormat = /\.(mp4|webm|ogg)$/i.exec(config.settings.customization.backgroundImageUrl)?.[1];
```

Comment:
Can you extract this regex to a constant?

Reply:
Like this?

```ts
const BackgroundImage = () => {
  const { config } = useConfigContext();

  if (!config?.settings.customization.backgroundImageUrl) {
    return null;
  }

  // Check if the background image URL is a video
  if (GetVideoFormat(config.settings.customization.backgroundImageUrl)) {
    return BackgroundVideo(config.settings.customization.backgroundImageUrl, videoFormat);
  }

  return (
    <Global
      styles={{
        '.mantine-AppShell-root': {
          minHeight: '100vh',
          backgroundImage: `url('${config?.settings.customization.backgroundImageUrl}')`,
          backgroundPosition: 'center center',
          backgroundSize: config?.settings.customization.backgroundImageSize ?? 'cover',
          backgroundRepeat: config?.settings.customization.backgroundImageRepeat ?? 'no-repeat',
          backgroundAttachment: config?.settings.customization.backgroundImageAttachment ?? 'fixed',
        },
      }}
    />
  );
};


const GetVideoFormat = (video: string) => {
    return /\.(mp4|webm|ogg)$/i.exec(video)?.[1];
}
```

Reply:
Or like this?

```ts
...
const videoFormatRegex = /\.(mp4|webm|ogg)$/i;
const videoFormat = videoFormatRegex.exec(config.settings.customization.backgroundImageUrl)?.[1];
```

Reply:
He meant to extract it to an array as far as I can tell

Reply:
Now I'm even more confused :D 

```ts
const GetVideoFormat = (video: string) => {
  const supportedFormats = ['mp4', 'webm', 'ogg'];
  for(const format of supportedFormats) {
    if(video.endsWith(format)) return format;
  }
}
```